File: 프로그래머스/lv0/120842. 2차원으로 만들기/2차원으로 만들기.java
class Solution {
    public int[][] solution(int[] num_list, int n) {
        int num=0;
        int[][] arr=new int[num_list.length/n][n];
    
    for(int i=0;i<num_list.length/n;i++){
        for(int j=0;j<n;j++){
            arr[i][j]=num_list[num];
            num++;
            }
        }
        return arr;
    }
}

File: 프로그래머스/0/120842. 2차원으로 만들기/2차원으로 만들기.py
def solution(num_list, n):
    answer = [[]]
    j = 0
    for i in range(len(num_list)):
        if len(answer[j]) < n:
            answer[j].append(num_list[i])
        else:
            answer.append([num_list[i]])
            j += 1
    return answer